rocks-lib/Cargo.toml

```diff
@@ -49,6 +49,7 @@ thiserror = "2.0.0"
 gpgme = "0.11.0"
 futures = "0.3.31"
 async-recursion = "1.1.1"
+predicates = "3.1.2"
```

Comment:
issue: I only see this used in tests. Should this be part of `dev-dependencies` instead?